src/languages/en.ts

```diff
@@ -808,12 +818,18 @@ export default {
         keepAll: 'Keep all',
         confirmApprove: 'Confirm approval amount',
         confirmApprovalAmount: 'Approve only compliant expenses, or approve the entire report.',
-        confirmApprovalAllHoldAmount: ({transactionCount}: ConfirmHoldExpenseParams) =>
-            `${Str.pluralize('This expense is', 'These expenses are', transactionCount)} on hold. Do you want to approve anyway?`,
+        confirmApprovalAllHoldAmount: () => ({
+            zero: `This expense is on hold. Do you want to approve anyway?`,
+            one: `This expense is on hold. Do you want to approve anyway?`,
+            other: () => `These expenses are on hold. Do you want to approve anyway?`,
```

Comment:
If we pass the count value 0 for the confirmApprovalAllHoldAmount key, it does not return the text for zero(`This expense is on hold. Do you want to approve anyway?`); it goes to other case(`These expenses are on hold. Do you want to approve anyway?`).

Reply:
So, why have we added the zero string here if it will never be used?

Reply:
Ohhkay, then I think we can remove that unnecessary part. I will finish the testing today, and I still need to test cases for `src/libs/Localize/index.ts`, as this file is a little complicated and used throughout the app.